stock_picking_validate/wizard/stock_picking_validate.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockPickingValidate(models.TransientModel):
+    _name = 'stock.picking.validate'
+
+    def action_stock_picking_validate(self):
+        context = dict(self._context or {})
+        active_ids = context.get('active_ids', [])
+        picking_ids = self.env['stock.picking'].browse(active_ids)
```

Comment:
We may insert a check on picking type - show error if there is more than one picking type involved.  My concern is that someone could make a mistake and validate a bunch of random pickings.  This check will not be perfect, but should reduce the risk a bit.  Can you think of any other reasonable measures?